classes/author/Author.inc.php

```diff
@@ -10,14 +10,14 @@
  * @class Author
  * @ingroup monograph
  *
- * @see AuthorDAO
+ * @see author\DAO
```

Comment:
This can be just `@see DAO` since they share a namespace.

Reply:
Fixed